ledger/block/src/bytes.rs

```diff
@@ -42,6 +42,20 @@ impl<N: Network> FromBytes for Block<N> {
         // Read the solutions.
         let solutions: Solutions<N> = FromBytes::read_le(&mut reader)?;
 
+        let prior_solution_ids = if version >= 2 {
+            // Read the number of prior solution IDs.
+            let num_prior_solutions = u32::read_le(&mut reader)?;
+            // TODO: we might want to restrict this value
+            // Read the aborted transaction IDs.
```

Comment:
```suggestion
            // Read the prior solution IDs.
```

An upper-bound check of `max_aborted_solutions` would be sufficient here for the TODO